openmc/polynomial.py

```diff
@@ -75,7 +75,68 @@ def order(self):
     def __call__(self, r):
         import openmc.lib as lib
         if isinstance(r, Iterable):
-            return [np.sum(self._norm_coef * lib.calc_zn_rad(self.order, r_i))
+            return [np.sum(self._norm_coef * lib.calc_zn_rad(self.order, r_i / self.radius))
```

Comment:
In the docstring for the class, right now we have `r` under "Parameters", which is really supposed to be the parameters for the `__init__` method (just `coef` and `radius`). Can you remove the `r` parameter and its description (especially since it doesn't match this change)?